apps/wallet-ui/next.config.mjs

```diff
@@ -1,4 +1,18 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  webpack: (config) => {
+    config.externals.push("pino-pretty", "lokijs", "encoding");
+    return config;
+  },
+  images: {
+    dangerouslyAllowSVG: true,
+    remotePatterns: [
+      {
+        protocol: "https",
+        hostname: "**",
+      },
+    ],
```

Comment:
we _cannot_ do this this means we're hosting an open to everyone image optimization proxy.

Reply:
Any alternatives? We can't assume all NFT data is on IPFS

Reply:
If not this we just can't use `<Image />`

Reply:
yep just use `<img />`